src/NUnitEngine/nunit.engine/Runners/ProcessRunner.cs

```diff
@@ -80,7 +80,7 @@ public override TestEngineResult Explore(TestFilter filter)
             }
             catch (Exception e)
             {
-                log.Error("Failed to run remote tests {0}", e.Message);
```

Comment:
The changes in `ProcessRunner` are just to log additional information - these log calls were always formatted correctly.